CMakeLists.txt

```diff
@@ -61,7 +68,7 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm64|arm64v8")
     if(${OQS_DIST_BUILD})
         set(OQS_DIST_ARM64_V8_BUILD ON)
     endif()
-elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "armel|armhf|armv7|arm32v7")
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "armel|armhf|armv7|arm32v7|arm")
```

Comment:
Is "arm" a valid (unambiguous) name for 32bit ARM? Is there a list of generally agreed-upon names for ARM architectures?

Reply:
Based on my (limited) personal experience, bare-metal and RTOS targets typically define 32bit ARM only as „arm“, 64bit ARM as „arm64“. But I will google about that to find a proper valid statement.

Reply:
“arm” is, in fact, not an unambiguous name for 32-bit ARM architectures (I found some systems using it also for 64-bit systems). Hence, I removed this option again and instead added a “workaround” for Zephyr within the CMakeLists.txt in the zephyr directory.